tests/converters/app-converter.test.ts

```diff
@@ -114,6 +114,7 @@ describe('App Converter', () => {
                     examplebucketPolicyE09B485E: {
                         Type: 'AWS::S3::BucketPolicy',
                         Properties: {
+                            PolicyDocument: {},
```

Comment:
These updates to the tests are required because it now validates required properties.